comms/src/blazingdb/transport/Server.cc

```diff
@@ -216,7 +216,7 @@ void ServerTCP::Run() {
 					this->putMessage(message->metadata().contextToken, message);
 				}
 			} catch(const std::runtime_error & exception) {
-				std::cerr << "[ERROR] " << exception.what() << std::endl;
+				std::cerr << "[ERROR] " << exception.what() << '\n';
```

Comment:
cerr doesnt get buffered so I am not sure removing the flush here actually helps.